pep-0621.rst

```diff
@@ -29,14 +29,15 @@ Motivation
 The key motivators of this PEP are:
 
 - Encourage users to specify core metadata statically for speed,
-  ease of specifying, and deterministic consuming
+  ease of specifying, deterministic consuming by build back-ends,
```

Comment:
```suggestion
  ease of specifying, deterministic consumption by build back-ends,
```

Might just be my regional dialect, but "consuming" as a noun like this doesn't sound right to me, I would use "consumption".

Reply:
```suggestion
  ease of specification, deterministic consumption by build back-ends,
```

Agreed. I'd also use "specification" - for the same reason.